views/includes/css.hbs

```diff
@@ -1,4 +1,4 @@
-{{!-- Copyright 2015-2016, Google, Inc.
+/* Copyright 2015-2016, Google, Inc.
```

Comment:
What was the reason for changing this? (Don't think it matter much either way, just curious.)